docs/gettingstarted/index.md

```diff
@@ -14,12 +14,13 @@ From Zero to Modding
     * the `gradle` folder
 3. Move the files listed above to a new folder, this will be your mod project folder.
 4. Choose your IDE:
-    * Forge explicitly supports developing with Eclipse or IntelliJ environments, but any environment, from Netbeans to vi/emacs, can be made to work.
+    * Forge explicitly supports developing with Eclipse, IntelliJ or Visual Studio Code environments, but any environment, from Netbeans to vi/emacs, can be made to work.
```

Comment:
Forge does not officially provide support for developing in Visual Studio Code. Although the run task exists, it is expected that users who wish to use VS Code for making Forge mods must solve any issues that arise from their use of VS Code by themselves (or through other community members).

Please revert this change, and add an admonition after the `genVSCodeRuns` paragraph to warn users of this.
(See the `Customizing Your Mod Information` section for an example of an admonition, the `!!! important` part)